### PR TITLE
Adds guidelines for contributing and generating documentation

### PR DESCRIPTION
**Pull request by duarteocarmo**
Adds guidelines on how to contribute more easily to documentation. 

Requested on #388 right [here](https://github.com/nteract/papermill/issues/386#issuecomment-505940465). 

File edited: [`CONTRIBUTING.MD`](https://github.com/nteract/papermill/blob/master/CONTRIBUTING.md)

The section added contains:
- how to generate docs
- a bit about what they use to be generated
- how to run tests on them

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ _Note: When you are finished you can use `source deactivate` to go back to your
 
 ### Running Tests Locally
 
+If you are contributing with documentation please jump to the [next section.](#Running-Documentation-Tests)
```
Comment (rgbkrk) [inline CONTRIBUTING.md]: I'd switch the wording up here:

```suggestion
If you are contributing with documentation please jump to [building documentation.](#Building-Documentation)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -69,6 +71,18 @@ The `pyargs` option allows `pytest` to interpret arguments as python package nam
 
 Now there should be a working and editable installation of Papermill to start making your own contributions.
 
+### Running Documentation Tests
+
+The documentation is build using the [Sphinx](http://www.sphinx-doc.org/en/master/) engine. To contribute, you can edit the `.rst` files (learn more about restructured text [here](https://en.wikipedia.org/wiki/ReStructuredText)).
```
Comment (rgbkrk) [inline CONTRIBUTING.md]:
```suggestion
The documentation is built using the [Sphinx](http://www.sphinx-doc.org/en/master/) engine. To contribute, edit the [RestructuredText (`.rst`)](https://en.wikipedia.org/wiki/ReStructuredText) files to make changes and additions.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -69,6 +71,18 @@ The `pyargs` option allows `pytest` to interpret arguments as python package nam
 
 Now there should be a working and editable installation of Papermill to start making your own contributions.
 
+### Running Documentation Tests
```
Comment (rgbkrk) [inline CONTRIBUTING.md]:
```suggestion
### Building Documentation
```

**Comment by rgbkrk** [comment]
Made a few suggested edits to improve how the links navigated as well as call out the documentation setup as "building" the documentation rather than testing.

**Comment by rgbkrk** [comment]
Thank you for all these quick additions to the project. Welcome aboard @duarteocarmo!

**Comment by duarteocarmo** [comment]
Accepted your suggestions 😊

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -69,6 +71,18 @@ The `pyargs` option allows `pytest` to interpret arguments as python package nam
 
 Now there should be a working and editable installation of Papermill to start making your own contributions.
 
+### Building Documentation
+
+The documentation is built using the [Sphinx](http://www.sphinx-doc.org/en/master/) engine. To contribute, edit the [RestructuredText (`.rst`)](https://en.wikipedia.org/wiki/ReStructuredText) files to make changes and additions.
```
Comment (willingc) [inline CONTRIBUTING.md]:
```suggestion
The documentation is built using the [Sphinx](http://www.sphinx-doc.org/en/master/) engine. To contribute, edit the [RestructuredText (`.rst`)](https://en.wikipedia.org/wiki/ReStructuredText) files in the docs directory to make changes and additions.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -69,6 +71,18 @@ The `pyargs` option allows `pytest` to interpret arguments as python package nam
...
+
+Once you are done editing, to generate the documentation, just run the documentation specific tests.
```
Comment (willingc) [inline CONTRIBUTING.md]:
```suggestion
Once you are done editing, to generate the documentation, use tox and the following command from the root directory of the repository:
```

**Comment by duarteocarmo** [comment]
Should’ve been clearer, thanks for the good suggestions!

**Pull request merged**